### PR TITLE
feat: review airgapped install (#276)

### PR DESCRIPTION
**Pull request by Piotr1215**
Closes DOC-276

[Air-Gapped Install Preview](https://deploy-preview-303--vcluster-docs-site.netlify.app/docs/platform/install/advanced/air-gapped)

### CODE REVIEW

File: platform/install/advanced/air-gapped.mdx [hunk 1]
```diff
@@ -2,36 +2,41 @@
...
-To solve not being able to pull images from Docker Hub, you can
-[use a private image registry](#offline-images) to host your own copies of the vCluster Platform images that
-your cluster can reach.
+To address image pulling issues from Docker Hub, you can [use a private image
+registry](#offline-images) to host your own copies of the platform images
+accessible to your cluster.
 
 ### Prerequisites
 
 - A running air-gapped Kubernetes cluster `v1.26.x`, `v1.27.x`, or `v1.28.x`
```
Comment (deniseschannon) [inline platform/install/advanced/air-gapped.mdx:29]: These versions are out of date. I'm wondering how we should handle this as a whole.

File: platform/_partials/install/upgrade.mdx [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-Upgrade vCluster Platform via:
+## Upgrade the platform
```
Comment (deniseschannon) [inline platform/_partials/install/upgrade.mdx:4]: Why are we using this section to upgrade the platform in the install docs? Why isn't this being used to install the platform?

Comment (deniseschannon) [reply platform/_partials/install/upgrade.mdx:4]: As a new user this doesn't make sense to me.

**Review by deniseschannon — APPROVED** [review]

File: platform/install/advanced/air-gapped.mdx (hunk 1 above)
Comment (Piotr1215) [reply platform/install/advanced/air-gapped.mdx:29]: Capturing for later: https://linear.app/loft/issue/DOC-301/update-kubernetes-versions-requirements-for-air-gapped

File: platform/_partials/install/upgrade.mdx (hunk 2 above)
Comment (Piotr1215) [reply platform/_partials/install/upgrade.mdx:4]: Capturing for later: https://linear.app/loft/issue/DOC-301/air-gapped-install-improvements

**Pull request merged**